### PR TITLE
chore(forge-wizard): to work with ngx-forge dependency

### PR DESCRIPTION
**Pull request by corinnekrych**
Fixes https://github.com/openshiftio/openshift.io/issues/1112

### CODE REVIEW

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2151-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2151-fabric8-ui.badger.fabric8.io

**Comment by corinnekrych** [comment]
@edewit could you review?
left-over components could be done as part of https://github.com/openshiftio/openshift.io/issues/1186
Shall we merge that one?

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2151-3 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2151-fabric8-ui.badger.fabric8.io

**Comment by joshuawilson** [comment]
I'll review once @edewit reviews it.

File: src/app/space/forge-wizard/components/pipeline-view/pipeline-view.component.ts [hunk 1]
```diff
@@ -76,6 +76,10 @@ export class PipelineViewComponent implements OnInit {
     choice.name = choice.id;
     choice.description = choice.descriptionMarkdown;
     choice.description = choice.description.replace(/\n\n/g, '\n');
+    // choice.description = marked(choice.description, (err, parseResult) => { console.log(err); });
+    // if (choice.stages && choice.stages[0] && !choice.stages[0].name) { // deal with back button format ouput only once.
+    //   choice.stages = this.buildStages(choice);
+    // }
```
Comment (edewit) [inline src/app/space/forge-wizard/components/pipeline-view/pipeline-view.component.ts]: better remove these

**Comment by edewit** [comment]
@joshuawilson can we merge this?

**Comment by fabric8cd** [comment]
@corinnekrych fabric8/fabric8-ui:SNAPSHOT-PR-2151-4 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2151-fabric8-ui.badger.fabric8.io

**Pull request merged**